source/winVersion.py

```diff
@@ -58,7 +55,9 @@ def _getRunningVersionNameFromWinReg() -> str:
 			try:
 				releaseId = winreg.QueryValueEx(currentVersion, "ReleaseID")[0]
 			except OSError:
-				releaseId = ""
+				raise RuntimeError(
+					"Release name is not recorded in Windows Registry on this version of Windows"
+				) from None

```

Comment:
Eh? Where did this come from?

Reply:
This is a pretty common way of raising an exception from the `except` block without original exception being re-raised. If this would not be done `RuntimeError` would be raised as expected but before that all FileNotFoundError's from the `winreg` module would be raised as well.